src/components/Layout/UserDropdown/index.tsx

```diff
@@ -4,8 +4,11 @@ import { useUser } from '../../../hooks/useUser';
 import axios from 'axios';
 import useClickOutside from '../../../hooks/useClickOutside';
 import { defineMessages, FormattedMessage } from 'react-intl';
+import Link from 'next/link';
 
 const messages = defineMessages({
+  myprofile: 'My Profile',
+  settings: 'User Settings',
```

Comment:
```suggestion
  myprofile: 'Profile',
  settings: 'Settings',
```

Or, "My Profile" and "My Settings" (I just think we should be consistent).

Reply:
Already changed this locally. Just forgot to push it heh.